test/functorch/test_aotdispatch.py

```diff
@@ -2590,6 +2590,7 @@ def test_aot_autograd_symbolic_exhaustive(self, device, dtype, op):
     torch.nn.GaussianNLLLoss,  # NotImplementedError: local_scalar_dense/item NYI for torch.bool
     torch.nn.CrossEntropyLoss,  # Cannot call sizes() on tensor with symbolic sizes/strides
     torch.nn.Bilinear,  # Cannot call sizes() on tensor with symbolic sizes/strides
+    torch.nn.MultiheadAttention,  # baddbmm - Cannot call sizes() on tensor with symbolic ...
```

Comment:
Badmm was captured in SDPA but now since there exists a path we are calling it in we need to skip